gui.py

```diff
@@ -47,17 +51,21 @@ def add_features(self, window, gui_save):
     def wav_file_location_handler(self):
         self.input_location, _ = QtWidgets.QFileDialog.getOpenFileName(self.window,
                                                                        "Choose input file", " ", "(*.wav)")
-        self.importFIleLabel.setText(f'file imported')
-        self.startButton.setDisabled(False)
+        if(self.input_location):
+            self.my_data.input_track_name = ''.join(self.input_location.split('/')[-1].split('.')[:-1])
+            self.importFIleLabel.setText(f'File imported')
```

Comment:
QA - fajnie byłoby wyświetlać tutaj nazwę pliku; użytkownik ma pewność że wybrał prawidłowy. To samo przy ekstrakcji.